manifests/prombench/stateful-set.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: prombench-test-{{ .PR_NUMBER }}
   labels:
     app: prombench-test-{{ .PR_NUMBER }}
+  annotations:
+    prometheus.io/prombench.exit_on_error: 'true'
+    prometheus.io/prombench.wait_on_update: 'true'
+    prometheus.io/prombench.retry_count: '90'
```

Comment:
again I think best to set this as a cli arg

Reply:
I think keeping `retry_count` as an annotation makes more sense because we might want different resources to have different retry count. having it as a cli arg will mean the same retry count will be enforced on all of the manifest files, similar to what `GlobalRetryCount` is currently doing

Reply:
ok in that case can add it as a cli arg - `-retryCount=60 -retryCountResourceName(or retryCountResourceFilename)=manifesFilename.yaml` or some similiar aproach.

my main concern here is that if we start setting options from different places it will get confusing.